src/PlaywrightSharp.Chromium/ChromiumBrowser.cs

```diff
@@ -168,12 +177,12 @@ public Task<string> StopTracingAsync()
 
         internal static async Task<IBrowser> ConnectAsync(IBrowserApp app, ConnectOptions options)
         {
-            var transport = await BrowserHelper.CreateTransportAsync(options).ConfigureAwait(false);
-            var connection = new ChromiumConnection(transport);
-            var response = await connection.RootSession.SendAsync(new TargetGetBrowserContextsRequest()).ConfigureAwait(false);
-            var browser = new ChromiumBrowser(app, connection, response.BrowserContextIds);
-            await connection.RootSession.SendAsync(new TargetSetDiscoverTargetsRequest { Discover = true }).ConfigureAwait(false);
-            await browser.WaitForTargetAsync(t => t.Type == TargetType.Page).ConfigureAwait(false);
+            var transport = await BrowserHelper.CreateTransportAsync(options: options).ConfigureAwait(continueOnCapturedContext: false);

```

Comment:
do we really need the parameters names everywhere here?

Reply:
ouch, I'm trying Rider 😂